src/app/audiologist-summary/audiologist-summary.component.ts

```diff
@@ -0,0 +1,231 @@
+import { Component, OnInit } from '@angular/core';
+import { Utilities } from '../common/utlilities';
+import { ThsDataService } from '../services/ths-data.service';
+import { TsScreenerDataService } from '../services/ts-screener-data.service';
+import { TfiDataService } from '../services/tfi-data.service';
+import { TsScreenerAnswerStrings, ThsAnswerStrings } from '../common/custom-resource-strings';
+import { TestsDataService } from '../services/tests-data.service';
+import { Subscription } from 'rxjs/Subscription';
+
+@Component({
+  selector: 'audiologist-summary',
+  templateUrl: './audiologist-summary.component.html',
+  styleUrls: ['./audiologist-summary.component.css']
+})
+export class AudiologistSummaryComponent implements OnInit {
+  public patientID;
+  public ts: string;
+  public thsA: number; public thsB: number;
+  public thsC: number; public thsCtxt: string; public thsCex: string;
+  public overallTFI: number; public intrusive: number; public sense: number; public cognitive: number;
+  public sleep: number; public auditory: number; public relaxation: number; public quality: number; public emotional: number;
+
+  public audtype: string; public leftHighSeverity: string; public leftHighConfig: string; public leftLowSeverity: string;
+  public leftLowConfig: string; public rightHighSeverity: string; public rightHighConfig: string;
+  public rightLowSeverity: string; public rightLowConfig: string; public ototype: string; public tymptype: string;
+
+  private subscription: Subscription;
+
+  /**
+   *
+   * @param thsDataService the data service for ths questionare
+   * @param tsDataService the data service for ts questionare
+   * @param tfiDataService the date service for tfi questionare
+   */
+  constructor(public thsDataService: ThsDataService, public tsDataService: TsScreenerDataService, public tfiDataService: TfiDataService,
+    public testsDataService: TestsDataService) {
+    this.tsDataService.onInit();
+    this.setTS();
+    this.thsDataService.onInit();
+    this.setTHS();
+    this.tfiDataService.onInit();
+    this.setTFI();
+    this.testsDataService.onInit();
+    this.updateTestResults(this.testsDataService.data);
+    this.patientID = Utilities.getSessionStorage('patient-id');
+  }
+
+  ngOnInit() {
+    this.subscription = this.testsDataService.observableData.subscribe(data => this.updateTestResults(data));
+  }
+
+  public setTS() {
+    let answers = this.tsDataService.dataRecord;
+    let tsAnswers = new TsScreenerAnswerStrings();
+    let index = 0;
+    if (answers.length < 1) {
+      return;
+    }
+    if (answers[index++].choice === tsAnswers.NO) {
+      this.ts = "No Tinnitus";
```

Comment:
In Typescript style Strings are represented by ' instead of quotation marks.
```suggestion
      this.ts = 'No Tinnitus';
```

Reply:
Thanks, I'm making this change.